tests/cpydiff/syntax_assign_expr.py

```diff
@@ -1,8 +1,8 @@
 """
 categories: Syntax,Operators
-description: MicroPython allows using := to assign to the variable of a comprehension, CPython raises a SyntaxError.
-cause: MicroPython is optimised for code size and doesn't check this case.
-workaround: Do not rely on this behaviour if writing CPython compatible code.
+description: MicroPython allows := to assign to the iteration variable of a nested comprehension, CPython does not.
```

Comment:
This sounds like it allows assigning to the *inner* comprehension variable, whereas it's the outer one that it wrongly allows.

Suggest "of a nested comprehension" -> "in nested comprehensions", which makes it a bit more of a general statement.  The full description which much clearer is given on the next line in the "cause".

Reply:
Updated!